UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -199,16 +199,16 @@ namespace UnitsNet
         public static $unitEnumName BaseUnit => $unitEnumName.$baseUnitSingularName;
 
 "@; 
-        if($quantity.BaseDimensions)
-        {
+    if($quantity.BaseDimensions)
+    {
 @"
-          /// <summary>
-          ///     The <see cref="BaseDimensions" /> of this quantity.
-          /// </summary>
-          public static BaseDimensions BaseDimensions => new BaseDimensions($baseDimensionLength, $baseDimensionMass, $baseDimensionTime, $baseDimensionElectricCurrent, $baseDimensionTemperature, $baseDimensionAmountOfSubstance, $baseDimensionLuminousIntensity);
+        /// <summary>
+        ///     The <see cref="BaseDimensions" /> of this quantity.
+        /// </summary>
+        public static BaseDimensions BaseDimensions => new BaseDimensions($baseDimensionLength, $baseDimensionMass, $baseDimensionTime, $baseDimensionElectricCurrent, $baseDimensionTemperature, $baseDimensionAmountOfSubstance, $baseDimensionLuminousIntensity);
```

Comment:
Use `BaseDimensions { get; } = new ...` instead to avoid allocating a new instance every time the property is called.